WhatsNext.html

```diff
@@ -86,98 +96,90 @@ <h3>The Queue</h3>
         The table below details our current plan for loading data to PROD. This table includes the state and
         date we plan to load the data.
       </p>
+      <h5 id="loadDataQueue"> Table 1: Load Data Queue</h5>
```

Comment:
No change required here, but I recommend merging in master one more time now that new h5 CSS has been added. We can see how it looks with your updated table.

Reply:
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202350/49e3c57b-a2b2-427a-b7fb-a9fdd9e472fc)